src/OpenAi.php

```diff
@@ -63,6 +63,38 @@ public function classification($opts)
         return $this->sendRequest($url, 'POST', $opts);
     }
 
+    /**
+     * @param $file
+     * @return bool|string
+     */
+    public function uploadFile($file)
+    {
+        $url = Url::filesUrl();
+
+        $key = $this->headers[1];
+        $this->headers = [
+            "Content-Type: multipart/form-data",
+            $key,
+        ];
```

Comment:
Seem cleaner isn't? By the way, I didn't test but should be work.
```suggestion
        $this->headers[0] = "Content-Type: multipart/form-data";
```